tests/integration/test_charm.py

```diff
@@ -64,6 +65,92 @@ async def test_flask_webserver_timeout(
             )
 
 
+@pytest.mark.parametrize(
+    "update_config, excepted_config",
+    [
+        pytest.param({"flask_env": "testing"}, {"ENV": "testing"}, id="env"),
+        pytest.param(
+            {"flask_permanent_session_lifetime": 100},
+            {"PERMANENT_SESSION_LIFETIME": 100},
+            id="permanent_session_lifetime",
+        ),
+        pytest.param({"flask_debug": True}, {"DEBUG": True}, id="debug"),
+    ],
+    indirect=["update_config"],
+)
+@pytest.mark.usefixtures("update_config")
+async def test_flask_config(flask_app, get_unit_ips, excepted_config):
```

Comment:
Would the arguments be better if type hinted? :) same for other tests

Reply:
Updated, thanks!